playbooks/roles/tower/tasks/bootstrap.yml

```diff
@@ -2,6 +2,18 @@
 # Authenticate with tower
 - include_tasks: authenticate_tower.yml
 
+- name: Upload license file "https://{{ tower_host }}"
+  uri:
+    url: "https://{{ tower_host }}/api/v2/config/"
+    return_content: yes
+    user: "{{ tower_username }}"
+    password: "{{ tower_password }}"
+    method: POST
+    force_basic_auth: yes
+    body_format: json
+    body: '{{ tower_license }}'
```

Comment:
We'll need to add in `validate_certs: '{{ tower_verify_ssl }}'` to this block to support self signed certs on PDS

Reply:
👍 Done